lib/api/core/models/repositories/userRepository.js

```diff
@@ -77,6 +78,12 @@ class UserRepository extends Repository {
       user._id = uuid.v4();
     }
 
+    if (user._id === this.anonymous()._id) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)